### ISSUE TITLE
Add ability to trigger a run using existing configuration

### ISSUE THREAD

**Issue by petems** [hashicorp/tfe-cli#24]
Maybe I'm missing something, but I couldn't figure out a way to trigger a plan on a workspace with the existing configuration? 

Right now I'm doing it with this code:

```
trigger_run () {
  export RES=$(curl -0 -X POST "$TFEAPI/runs" \
  --header "Authorization: Bearer $TFE_TOKEN" \
  --header "Content-Type: application/vnd.api+json" \
  --data @- << EOF
{
  "data": {
    "attributes": {
      "is-destroy":false,
      "message": "Triggered from Trigger all Shell Script"
    },
    "type":"runs",
    "relationships": {
      "workspace": {
        "data": {
          "type": "workspaces",
          "id": $1
        }
      }
    }
  }
}
EOF
)
```

Is this possible?

**Comment by fprimex** [comment]
You're correct that this ability isn't there. `tfe`'s original mandate was to replace `terraform push`, which only operated on local configurations rather than any configuration stored in TFE already. This is certainly part of the API and can be done, however I would like to wait on considering this for at least another week as there are some discussions about the CLI going on.

Thanks for your patience.

**Comment by fprimex** [comment]
I implemented this on top of the new `-destroy` parameter because I'm too lazy to make a new branch. Please see:

https://github.com/hashicorp/tfe-cli/tree/push-destroy

I make a `-current-config` option to `pushconfig`.

**Comment by fprimex** [comment]
merged into master
